webcheck.go

```diff
@@ -134,6 +134,12 @@ func (fm *Frontman) runWebCheck(data WebCheckData) (map[string]interface{}, erro
 	}
 
 	data.Method = strings.ToUpper(data.Method)
+
+	logrus.Debug("web request w timeout ", timeout)
```

Comment:
looks like debug message that is not intended to be left for production code.

Reply:
Indeed, thanks. Fixed